otter/scheduler.py

```diff
@@ -197,8 +197,9 @@ def execute_event(store, log, event, deleted_policy_ids):
     group_id = event['groupId']
     policy_id = event['policyId']
     log = log.bind(tenant_id=tenant_id, scaling_group_id=group_id,
-                   policy_id=policy_id)
-    log.msg('Scheduler executing policy {policy_id}')
+                   policy_id=policy_id,
+                   scheduled_time=event["trigger"].isoformat() + "Z")
+    log.msg('sch-exec-pol', cloud_feed=True)
```

Comment:
Should this be `'sch-exec-policy'`, since that's what's been registered in spec?